tests/test_readme.py

```diff
@@ -30,7 +30,8 @@
 from beamme.core.mesh import Mesh
 
 
-def extract_code_snippets() -> tuple[dict[str, str], list[str]]:
+@pytest.fixture(scope="session")
+def extract_code_snippets(repo_root: Path) -> tuple[dict[str, str], list[str]]:
```

Comment:
... just use 


```suggestion
def extract_code_snippets(pytestconfig: Any) -> tuple[dict[str, str], list[str]]:
```

here and ...

Reply:
Good point, that makes total sense, I remove the fixture and added `pytestconfig` here.